src/components/ui/navbar.tsx

```diff
@@ -23,14 +34,26 @@ const Navbar: React.FC = () => {
     },
   };
 
+  useEffect(() => {
+    if (pathname === "/") setActivePage(0);
+    else if (pathname === "/blog") setActivePage(1);
+    else if (pathname === "/tentang-kami") setActivePage(2);
+  }, [pathname]);
+
```

Comment:
please separate the logic from the page, maybe it can be changed into separate hooks like "useActivePage"